### PR TITLE
#415 skipBundlesTest option

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for #415 

Tested in IntelliJ: ``Run->Edit Configurations->VM Options``: ``-DskipBundlesTest``

Tested in terminal: 
- ``mvn clean install`` -> 1m 30s
- ``mvn clean install -DskipBundlesTest`` -> 40s

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#786](https://github.com/zerocracy/farm/issues/786) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#786](https://github.com/zerocracy/farm/issues/786) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/681924641). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

File: src/test/java/com/zerocracy/BundlesTest.java [hunk 1]
```diff
@@ -125,6 +127,14 @@
 
     private Path home;
 
+    @BeforeClass
+    public static void maybeSkipThem() {
+        Assume.assumeTrue(
+            "Parameter skipBundlesTest found, skipping...",
+            System.getProperty("skipBundlesTest") == null
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/BundlesTest.java]: @amihaiemil perhaps we can use `Matchers.nullValue` here

File: src/test/java/com/zerocracy/BundlesTest.java [hunk 2]
```diff
@@ -125,6 +127,14 @@
 
     private Path home;
 
+    @BeforeClass
+    public static void maybeSkipThem() {
```
Comment (carlosmiranda) [inline src/test/java/com/zerocracy/BundlesTest.java]: @amihaiemil the name is a bit confusing. Let's call it `checkShouldRun` or something.

File: src/test/java/com/zerocracy/BundlesTest.java (hunk 1 above)
Comment (amihaiemil) [reply src/test/java/com/zerocracy/BundlesTest.java]: @carlosmiranda If we use:

```java
MatcherAssert.assertThat(
     "Parameter skipBundlesTest found, skipping...",
    System.getProperty("skipBundlesTest"),
    Matchers.nullValue()
);
```

Then the tests will fail. With ``Assume``, the exception will be thrown but the tests will "pass", will be ignored.

Comment (carlosmiranda) [reply src/test/java/com/zerocracy/BundlesTest.java]: @amihaiemil I don't mean using assert. I mean this: http://junit.sourceforge.net/javadoc/org/junit/Assume.html#assumeThat(T, org.hamcrest.Matcher). Can it work here?

```java
Assume.assumeThat(System.getProperty("skipBundlesTest"), Matchers.nullValue())
```

**Comment by amihaiemil** [comment]
@carlosmiranda right, it seems to work fine like this. Thanks :)

**Comment by carlosmiranda** [comment]
@yegor256 good to merge here

**Comment by yegor256** [comment]
@amihaiemil how about documenting this in README? Otherwise nobody will know about this feature.

**Comment by amihaiemil** [comment]
@yegor256 done :)

**Comment by yegor256** [comment]
@rultor merge

**Comment by carlosmiranda** [comment]
@0crat ping

**Comment by 0crat** [comment]
> @0crat ping ([here](https://github.com/zerocracy/farm/issues/786#issuecomment-372522374))

@carlosmiranda Can't understand "ping", try one of these:
  - `assign` Assign a performer to this issue
  - `boost` Set a boost for the job
  - `hello` Just say hello
  - `in` Register this issue in scope (WBS)
  - `out` Close the order and remove this job from scope
  - `pay` Pay a user some extra cash
  - `quality` Review a task
  - `resign` Resign from this issue
  - `status` Check the status of the job
  - `version` Print current version of the bot
  - `wait` Register an impediment for a job

**Comment by carlosmiranda** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/786#issuecomment-372557146))

@carlosmiranda This is what I know about this job, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#786](https://github.com/zerocracy/farm/issues/786) is not in scope
  * The job is waiting quality review verdict by @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)
  * Job [footprint](https://www.0crat.com/footprint/C3NDPUA8L?q=%7Bjob%3A%27gh%3Azerocracy%2Ffarm%23786%27%7D) (restricted area)

**Comment by ypshenychka** [comment]
@yegor256 Should I do QA review here or wait for 0crat to ask me?

**Comment by yegor256** [comment]
@ypshenychka yes, please go ahead. Zerocrat just fails to notify in GitHub sometimes.

**Comment by ypshenychka** [comment]
@carlosmiranda According to our [QA Rules](http://www.zerocracy.com/policy.html#42):
>The code reviewer found at least three problems in the code.
Comments were mostly about design problems, not cosmetic issues.

Only two issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed

**Comment by ypshenychka** [comment]
@carlosmiranda thanks

**Comment by ypshenychka** [comment]
@0crat quality acceptable

**Comment by 0crat** [comment]
Order was finished, quality is "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
Quality review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**